emsdk_manifest.json

```diff
@@ -633,6 +633,7 @@
     "bitness": 64,
     "url": "https://github.com/juj/ccache.git",
     "git_branch": "emscripten",
+    "activated_cfg": "CCACHE=%installation_dir%/bin",
```

Comment:
CCACHE is not a recognized setting the emscripten config file as far as I can tell.

The valid config settings are listed here: https://github.com/emscripten-core/emscripten/blob/7cdfa8d9f22562e7c08789a5ae29ad9a1025a8e4/tools/config.py#L122-L140  (not the `CACHE` is unrelated to ccache).

Reply:
Ah, should I add a CCACHE setting there then? FWIW, when I made this change locally, it worked fine without adding it to the config.py list.